### PR TITLE
Parameter description suggestion

### PR DESCRIPTION
**Pull request by PowershellScripts**
Suggested a parameter description

### CODE REVIEW

File: sharepoint/sharepoint-ps/sharepoint-online/Set-SPOTenantContentTypeReplicationParameters.md [hunk 1]
```diff
@@ -51,13 +51,14 @@ Tenant admin can use this cmdlet to select only some content types only for repl
...
 
-Tenant admin can  also reset to replicate all the content types.
+Tenant admin can also reset to replicate all the content types.
 
 ## PARAMETERS
 
 ### -ReplicateAllContentTypes
 
-{{Fill ReplicateAllContentTypes Description}}
+The ReplicateAllContentTypes switch specifies whether all content types should be replicated.
```
Comment (dariomws306) [inline sharepoint/sharepoint-ps/sharepoint-online/Set-SPOTenantContentTypeReplicationParameters.md]:
```suggestion
The ReplicateAllContentTypes parameter specifies whether all content types should be replicated.
```

File: sharepoint/sharepoint-ps/sharepoint-online/Set-SPOTenantContentTypeReplicationParameters.md [hunk 2]
```diff
@@ -51,13 +51,14 @@ Tenant admin can use this cmdlet to select only some content types only for repl
...
+* To allow replication for all content types, use this switch without a value.
```
Comment (dariomws306) [inline sharepoint/sharepoint-ps/sharepoint-online/Set-SPOTenantContentTypeReplicationParameters.md]:
```suggestion

```

**Comment by get-itips** [comment]
Hello @PowershellScripts 

Thanks for your contribution.
I know that the descriptions for the examples were present before your PR but, taking advantage of your PR, can I ask you to improve them?, for example:

`Tenant admin can also reset to replicate all the content types.`

to

`In this example a tenant admin can set to replicate all the content types.`

And something similar for the first Example.

Thank you 👍

**Comment by PowershellScripts** [comment]
@get-itips 

Do you want 
a) "This example sets.... blabla"
b) "Example 3 sets...."
c) "In this example a tenant admin/site collection administrator ...."


The description should be above or below the Powershell line?


If you have a decision, I could try to make the Docs consistent. 


> Hello @PowershellScripts
> 
> Thanks for your contribution.
> I know that the descriptions for the examples were present before your PR but, taking advantage of your PR, can I ask you to improve them?, for example:
> 
> `Tenant admin can also reset to replicate all the content types.`
> 
> to
> 
> `In this example a tenant admin can set to replicate all the content types.`
> 
> And something similar for the first Example.
> 
> Thank you

**Comment by get-itips** [comment]
Hello @PowershellScripts 
Thanks for returning, PR has already been merged so if you want you can open a new one.
I would say option c) but any of those will work.
The description should be below the example. (where the current description is) Thanks.

**Pull request merged**